storage/olddbclean/interface.go

```diff
@@ -0,0 +1,19 @@
+package olddbclean
```

Comment:
this sounds odd. Why not call it just "clean"?

Reply:
renamed